Gui/AboutWindow.cpp

```diff
@@ -98,6 +98,7 @@ AboutWindow::AboutWindow(QWidget* parent)
     ///filling tabs now
 
     _aboutText = new QTextBrowser(_tabWidget);
+    _aboutText->setObjectName( QString::fromUtf8("TextBrowser") );
```

Comment:
what is the objectname in Qt5? Doesn't it make more sense to use that objectname in the stylesheet as an alternative, rather than cluttering (a tiny bit) the code?

Reply:
For some reason QTextBrowser defaults to black text in some widgets, but not all. So adding a objectname makes it easy to style the widgets that needs fixes.

Can of course figure out a better solution, just launched Natron and noticed the issue. Will close and submit a new PR when changed.

Reply:
can you maybe print some properties of _aboutText to see why it didn't catch the right stylesheet entry? I know @acolwell cares a lot about code clutter ;-)
If that's the only way to go, then let's do it, but in this case I would ask for @acolwell's review as well

Reply:
A agree to avoid code clutter, will find a qss-only fix.